drivers/power/pm_register.c

```diff
@@ -62,19 +62,11 @@ int pm_register(FAR struct pm_callback_s *callbacks)
 
   /* Add the new entry to the end of the list of registered callbacks */
 
-  if (OSINIT_OS_READY())
```

Comment:
Why this condition is removed? I mean why not keeping it and changing to `if (!OSINIT_OS_INITIALIZING())`?

Reply:
Don't need.
Because when in system init state, we allow user to get lock.